docs/blog/2020-04-29-incredimental-builds/index.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "Gatsby Cloud Incremental Builds: More like *Incredimental* Builds!"
+title: "Incredimental Builds"
+seoTitle: "Gatsby's Incremental Builds: A Closer Look"
```

Comment:
Can we document this frontmatter field here: https://www.gatsbyjs.org/contributing/docs-and-blog-components/#blog-posts

Reply:
Definitely :+1: still trying to get the blasted thing working but will add this change on my next push.

Thanks for the heads up!